docs/source/optimizers.rst

```diff
@@ -1,7 +1,88 @@
 .. _optimizers:
 
+************
 Optimization
-===============
+************
+
+Lightning offers two modes for managing the optimization process:
+
+- automatic optimization (AutoOpt)
+- manual optimization
+
+For the majority of research cases, **automatic optimization** will do the right thing for you and it is what
+most users should use.
+
+For advanced/expert users who want to do esoteric optimization schedules or techniques, use **manual optimization**.
+
+------
+
+Manual optimization
+===================
+For advanced research topics like reinforcement learning, sparse coding, or GAN research, it may be desirable
+to manually manage the optimization process. To do so, do the following:
+
+* Ignore the optimizer_idx argument
+* So we can scale the loss automatically for you use self.backward(loss) instead of loss.backward()
+
```

Comment:
do the same caveats from before apply? can they be included in the docs?

Reply:
which caveats?